scripts/ktlint-pre-commit

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+echo "********************************"
+echo "Checking code formatting"
+echo "********************************"
+
+ktlint -F --android app/src/**/*.kt data/src/**/*.kt domain/src/**/*.kt testing/src/**/*.kt utility/src/**/*.kt
+
+status=$?
+
+if [ "$status" = 0 ] ; then
+  echo "Lint completed successfully."
+  exit 0
+else
+  echo "********************************"
+  echo "Ktlint issue found."
+  echo "Please fix the above issues. You could also use ktlint -F --android path/to/file.kt
+  command to fix most of the issues."
+  echo "Please note, there might be possibility where the above command will not fix the issue,
+  you have to fix it yourself."
```

Comment:
Change the above text to - "Please note, there mChight be a possibility where the above commend will not fix the issue. In that case, you will have to fix it yourself.

Reply:
Done, resolving on behalf of Vinita as it's a small fix.